### ISSUE TITLE
Help wanted / question : "exclude ALL but some" resources

### ISSUE THREAD

**Issue by alainvalain** [ebaauw/homebridge-hue#400]
Hello!

First of all, thank you @ebaauw for your work! Second, I hope this is an acceptable way to ask for some help.

**My challenge:** 
I would like to hide all resources linked to my Philips Hue bridge, EXCEPT one (or several) that I explicitly select/configure.

**Why?**
I have several Hue bulbs that I use through the Hue Bridge. I have one Osram Smart Plug that is also connected to the Hue bridge, but that doesn't show up on Homekit --> hence homebridge! 

My homebridge with homebridge-hue plugin causes to expose my Hue Bridge bulb resources twice to Homekit, once per connected bridge. I thus want to hide all ressources in homebridge EXCEPT this Osram switch, and this without having to individually block each and every bulb.

In a nutshell, I am looking how to implement a "deny all, allow some" rule.

Thanks a lot!

**Comment by ebaauw** [comment]
What's your config.json?  By default, homebridge-hue only exposes non-Hue lights connected to a v2 Hue bridge.

For the "allow some", use a whitelist `resourcelink`, see https://github.com/ebaauw/homebridge-hue#configuration.
